src/main/java/no/nav/veilarbperson/utils/DateUtils.java

```diff
@@ -0,0 +1,28 @@
+package no.nav.veilarbperson.utils;
+
+import java.sql.Timestamp;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Optional;
+
+public class DateUtils {
+
+    public static SimpleDateFormat getISODateFormatter() {
+        SimpleDateFormat simpleDateFormat = new SimpleDateFormat("yyyy-MM-dd");
+        simpleDateFormat.setLenient(false);
+        return simpleDateFormat;
+    }
+    public static Timestamp dateToTimestamp(Date date) {
+        return Optional.ofNullable(date).map(Date::toInstant).map(Timestamp::from).orElse(null);
+    }
+    public static Timestamp getTimestampFromSimpleISODate(String simpleISODate) {
```

Comment:
Her kan man bruke LocalDateTime i stede for sql Timestamp:
`LocalDateTime.parse(simpleISODate, ISO_LOCAL_DATE)`